src/style.css

```
@@ -1,3 +1,68 @@
-body {
+
+body,
+p,
+ul,
+li {
   margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Arial, sans-serif;
+  background-color: #f0f0f0;
+}
+
+header {
```

Write styles with classes selector
```suggestion
header {
```